src/java.base/share/classes/sun/security/util/DisabledAlgorithmConstraints.java

```diff
@@ -135,6 +136,7 @@ public DisabledAlgorithmConstraints(String propertyName,
             AlgorithmDecomposer decomposer) {
         super(decomposer);
         disabledAlgorithms = getAlgorithms(propertyName);
+        loadDisabledPatterns();
```

Comment:
You only need to call this method if `propertyName.equals("jdk.tls.disabledAlgorithms")`

Reply:
Good point, I'll add the check.